package.json

```diff
@@ -77,6 +77,7 @@
     "@tsconfig/recommended": "^1.0.2",
     "@typescript-eslint/eslint-plugin": "^5.48.2",
     "@typescript-eslint/parser": "^5.48.2",
+    "aws-cdk": "~2.133.0",
```

Comment:
since this can be executed outside of our npm scripts, I suggest doing a global install, like we have on `preinstall` - [link](https://github.com/metriport/metriport/blob/9eeca63a7bf5f5ce683dd89f4ef90939f0e0a108/package.json#L7)

Reply:
Great point, let's keep like this and move the ones on preinstall here, update the README/onboarding and notify the team?

Reply:
I think the ones in preinstall need to stay there because `is-ci` is used in prepare-dev which runs on `preinstall`, before dependencies get installed. And rimraf is used to nuke node_modules which would include nuking itself if it wasn't installed globally.

At least thats what it looks like to me